products/memgraph/upcoming/tutorials/marvel-universe.md

```diff
@@ -0,0 +1,165 @@
+## Marvel Universe Social Network
+
+### Introduction
+
+Spandex. Muscles. Big egos. Bad hair. No, we're not talking about your high
+school thrash metal band. We're talking one of the largest fictional social
+networks that is the Marvel Comic Universe! Here we'll teach you how to navigate
+this complex and confusing assembly of heroes and villains. If you've ever
+wanted to know who's Spider-Man's best super-buddy, or wanted to find all the
+comic issues where Hulk, Wolverine, Thor, and Black Panther appear together, look
+no further and fire up that Memgraph copy of yours!
+
+### Data Model
+
+Although the MCU is chock-full of heroes, the real hero here is Russ Chappell,
+who painstakingly gathered the MCU data for the [Marvel Chronology Project](www.chronologyproject.com).
+In addition, R. Alberich, J. Miro-Julia, and F. Rossello, three data scientists,
+scraped the Chronology Project database, processed the data and put it into a
+format that can be easily imported into any data-processing framework available
+today. Their aim was to investigate whether this fictional "social network" has
+a structure similar to a real-life social network. You can find their interesting
+findings in the paper that was the culmination of their work, linked [here](https://arxiv.org/pdf/cond-mat/0202174.pdf).
+The data they used, on the other hand, can be found [here](https://www.kaggle.com/csanhueza/the-marvel-universe-social-network).
+We've used a slightly modified version of this data to create a graph database
+snapshot ready for use.
+
+Now, the data we'll be using in our queries can be classified as follows:
+  * nodes, labeled as either "Hero" or "Comic"
+    * a "Hero" node has a "name" attribute corresponding to both a hero's
+      monniker and her/his real name (e.g. "SPIDER-MAN/PETER PARKER")
```

Comment:
I think that `monniker` is misspelled.
https://www.thefreedictionary.com/moniker

Reply:
Indeed. Will correct.